src/backend/marsha/core/simple_jwt/tokens.py

```diff
@@ -70,7 +70,7 @@ class ResourceAccessToken(AccessToken):
     """
 
     @classmethod
-    def for_resource(  # pylint: disable=too-many-arguments
+    def for_resource_id(  # pylint: disable=too-many-arguments
```

Comment:
`resource_id` should not be mandatory then ?

Reply:
For now it is optional, since the method can manage both "public" resource and LTI. The argument will be mandatory in a future PR to split LTI part in another method.

Reply:
:+1: